features/product/viewing_products/seeing_products_lowest_price_on_taxon_which_was_not_excluded.feature

```diff
@@ -0,0 +1,43 @@
+@viewing_products
+Feature: Seeing the product's lowest price on taxon which was not excluded
```

Comment:
```suggestion
Feature: Seeing the product's lowest price on taxon which was excluded from displaying the lowest price of discounted products or not
```
as this feature is for both excluded and not excluded taxons.

Reply:
oh, it doesn't make sense still 🙃

Maybe `Not seeing the product's lowest price...`? Because this is the feature we check here. We check exclusion so `not seeing` is more direct IMO.

Reply:
changed it as you requested, I agree the feature is not to display it and it could be misleading 🤔